requirements/base.in

```diff
@@ -2,7 +2,7 @@
 -c constraints.txt
 
 Django
-PyJWT
+pyjwt[crypto]>=2.1.0 # depends on newer jwt.decode and jwt.encode functionality
 six
 social-auth-app-django
-social-auth-core           # Common auth interfaces
+social-auth-core # Common auth interfaces
```

Comment:
Nit: I think the original spacing was so that all the comments could be aligned together and stand-out, but not sure if it is worth it or not.